cockroachcloud/migrate-from-serverless-to-dedicated.md

```diff
@@ -27,7 +27,7 @@ In {{ site.data.products.serverless }} clusters, all external service integratio
 {% include copy-clipboard.html %}
 ~~~ shell
 $ cockroach sql \
---url 'postgres://<username>:<password>@free-tier.<region>.cockroachlabs.cloud:26257?sslmode=verify-full&sslrootcert=<path/to/certs_dir>/cc-ca.crt&options=--cluster=<cluster_name>' \
+--url 'postgres://<username>:<password>@<cluster-name>.<region code>.cockroachlabs.cloud:26257?sslmode=verify-full&sslrootcert=<path/to/certs_dir>/cc-ca.crt' \
```

Comment:
is this right? Wouldn't it be the {host} not {cluster-name}?

Reply:
Yeah, good point. I'll change it to <host>.